.travis.yml

```diff
@@ -1,11 +1,11 @@
 language: node_js
-node_js: 
+node_js:
   - stable
   - lts/*
 before_install:
   - npm config set spin false --global
 install:
-  - yarn --ignore-scripts # ignore prepublish scripts
+  - yarn --frozen-lockfile --ignore-scripts # ignore prepublish scripts
```

Comment:
I recall that `yarn` had a `--ci` flag, but I can't find docs on it any more. This example is linked as the recommended style (https://codefresh.io/docs/docs/learn-by-example/nodejs/react/). Can we maybe try setting the environment variable `CI=true`?

Reply:
Doesn’t most CI set CI to true for you?

https://docs.travis-ci.com/user/environment-variables/#convenience-variables

Reply:
Yarn docs:

> If you need reproducible dependencies, which is usually the case with the continuous integration systems, you should pass --frozen-lockfile flag

That’s the only mention of continuous integration on that page; I believe it’s the currently recommended way.